types/node/worker_threads.d.ts

```diff
@@ -85,7 +89,12 @@ declare module "worker_threads" {
         readonly threadId: number;
         readonly resourceLimits?: ResourceLimits;
 
-        constructor(filename: string, options?: WorkerOptions);
+        /**
+         * @param filename  The path to the Worker’s main script or module.
+         *                  Must be either an absolute path or a relative path (i.e. relative to the current working directory) starting with ./ or ../,
+         *                  or a WHATWG URL object using file: protocol. If options.eval is true, this is a string containing JavaScript code rather than a path.
+         */
+        constructor(filename: string | URL, options?: WorkerOptions);
```

Comment:
I think `URL` needs to be imported.

Reply:
We shouldn't have to, it's a global in node, I willl look into that but I feel like we've already had this issue.

Reply:
it's once again an issue with DOM.
Refs: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/25356
Refs: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/25342

Reply:
It's also that we cannot re-export types as globals...